SwiftUI/ReviewWQuestion-Calculator/CustomButton.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  CustomButton.swift
+//  ReviewWQuestion-Calculator
+//
+//  Created by KiWoong Hong on 2022/05/29.
+//
+
+import SwiftUI
+
+struct CustomButton: View {
+    
+    let buttonInfo: CalculatorButtonManager
+    @ObservedObject var vm: CalculatorViewModel
+    
+    var body: some View {
+        
+        let size = UIScreen.main.bounds
+        
+        Button(action: {
+            vm.input(button: buttonInfo)
+        } , label: {
```

Comment:
```suggestion
        }, label: {
```

콤마 앞에 띄어쓰기가 하나 더 들어가있는 것은 의도된 띄어쓰기일까요?
띄어쓰기, 들여쓰기가 일정하게 맞춰지면 더욱 코드 퀄리티가 높아보일 거 같습니다 😄

Reply:
👍